### PR TITLE
Clenup main

### PR DESCRIPTION
**Pull request by tadelesh**
Remove all track1 samples and refine docs

### CODE REVIEW

**Comment by tadelesh** [comment]
Do not merge now as the ci broke again.

File: sdk/resourcemanager/appservice/staticsite/README.md [hunk 1]
```diff
@@ -1,13 +1,3 @@
----
-page_type: sample
```
Comment (lirenhe) [inline sdk/resourcemanager/appservice/staticsite/README.md:2]: why do we remove this?

Comment (lirenhe) [inline sdk/resourcemanager/appservice/staticsite/README.md:2]: why do we remove this?

File: services/apps/tracing/main.go [hunk 2]
```diff
@@ -1,153 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
```
Comment (lirenhe) [inline services/apps/tracing/main.go:2]: why are those files deleted?

Comment (lirenhe) [inline services/apps/tracing/main.go:2]: why are those files deleted?

**Comment by tadelesh** [comment]
> Not sure why we should delete track 1 samples. Is this topic discussed? @xboxeer

Like the SDK repo, we also move the track1 samples to the `legacy` branch.

**Comment by xboxeer** [comment]
Do we know where these samples are referenced? only in API reference site?

**Comment by tadelesh** [comment]
> Do we know where these samples are referenced? only in API reference site?

Some docs will directly refer to this repo: https://github.com/Azure-Samples/azure-sdk-for-go-samples. So, I think for main, we should not show track1 samples.

File: sdk/resourcemanager/appservice/staticsite/README.md (hunk 1 above)
Comment (tadelesh) [reply sdk/resourcemanager/appservice/staticsite/README.md]: restored.

Comment (tadelesh) [reply sdk/resourcemanager/appservice/staticsite/README.md]: Seems it is used for doc index. I restored it.

File: services/apps/tracing/main.go (hunk 2 above)
Comment (tadelesh) [reply services/apps/tracing/main.go:2]: Track1 samples removed from main.

Comment (tadelesh) [reply services/apps/tracing/main.go:2]: Track1 samples removed from main.

**Comment by tadelesh** [comment]
@xboxeer Do you have any further concern about the change?

File: README.md [hunk 3]
```diff
@@ -9,39 +9,98 @@ description: "A collection of samples showing how to use the Azure SDK for Go."
 
 # Azure SDK for Go Samples
 
-azure-sdk-for-go-samples is a collection of sample usages of the current version of the [Azure/azure-sdk-for-go][][here](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/services). The current version of the [Azure/azure-sdk-for-go][] is referring the service packages under the `/services` directory.
+This repo contains a collection of sample usages of the new version of [Azure/azure-sdk-for-go][]. All the samples are placed under [`sdk`](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk) folder and the folder structures are corresponding to the service packages under the `/sdk` directory of the [Azure/azure-sdk-for-go][] repo.
```
Comment (xboxeer) [inline README.md:12]: are we missing a url link here?

Comment (xboxeer) [inline README.md:12]: Suggest add our Golang SDK Quick start at the beginning.

File: README.md [hunk 4]
```diff
@@ -9,39 +9,98 @@ description: "A collection of samples showing how to use the Azure SDK for Go."
...
+
+You will need Go 1.18 and latest version of resource management modules.
+
+You will need to authenticate to Azure either by using Azure CLI to sign in or setting environment variables.
+
+#### Using Azure CLI to Sign In
+
+You could easily use `az login` in command line to sign in to Azure via your default browser. Detail instructions can be found in [Sign in with Azure CLI](https://docs.microsoft.com/cli/azure/authenticate-azure-cli).
+
+#### Setting Environment Variables
```
Comment (xboxeer) [inline README.md:28]: If I use Azure CLI login to authenticate, the system env setting are not required I think, we can add a statement at the begin: If your code is going to run in a none-user-interactive environment, you can also authenticate by setting environment variables as below.

**Review by xboxeer — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (tadelesh) [reply README.md:12]: Nope. It will use the link defined in the end of the doc.

File: README.md (hunk 4 above)
Comment (tadelesh) [reply README.md:28]: Added.

**Review by xboxeer — APPROVED** [review]
looks all good to me

**Comment by tadelesh** [comment]
@xboxeer Could you help to confirm the deprecation message used in `go.mod` and `README.md`.
Deprecated: Please note, this package has been deprecated. Replacement can be found under [github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk/). We strongly encourage you to upgrade to continue receiving updates. See [Migration Guide](https://aka.ms/azsdk/golang/t2/migration) for guidance on upgrading. Refer to our [deprecation policy](https://azure.github.io/azure-sdk/policies_support.html) for more details.

**Comment by lirenhe** [comment]
@tadelesh,  what is this build failure about?
